duckduckgo/Android
# ADS: Maestro Tests

Task/Issue URL:  https://app.asana.com/0/1174433894299346/1203827857866586

### Description
Adds a Maestro test to open the ADS Preview pages whenever a change to a Component happens


### Steps to test this PR

_Non Android 13 device_
- [ ] Fresh install InternalRelease or InternalDebug version
- [ ] Run maestro tests locally (❯ maestro test .maestro/ads_preview_flows/1-_design-system-components.yaml)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203827857866586
